containers/base/javascript/browser/css/dashboard.css

```diff
@@ -194,14 +201,23 @@ a{
 }
 
 .dragover{
-    background-color: var(--premium-white-blue);
+    background-color: var(--premium-white-blue) !important;
 }
 
 .sending-interface .cardcontainer{
     display: flex;
     flex-wrap: wrap;
 }
 
+.action{
+    background-color: var(--premium-white) !important;
```

Comment:
`!important` really needed? 🙈

Reply:
Unfortunately needed to overrule material design.